src/modules/organizations/actions.js

```diff
@@ -73,18 +78,29 @@ export function create({name, plan}) {
 export function addMember(organizationId, email) {
   return (dispatch, getState) => {
     const cid = cuid()
-    let object = {id: cid, organization_id: organizationId, user_id: 4}
-
-    const action = oActions.createStart(object);
+    let object = {id: cid, organization_id: organizationId}
 
     api(getState()).organizations.addMember({organizationId, email}, (err, membership) => {
-      if (err) {
-        dispatch(oActions.createError(err, object))
-      }
-      else if (membership) {
+      if (!!membership) {
         dispatch(userActions.fetchSuccess([membership._embedded.user]))
         dispatch(membershipActions.createSuccess([membership]))
       }
     })
   }
 }
+
+export function addFact(organization, rawFact) {
+  return (dispatch, getState) => {
+    const cid = cuid()
+    let object = {id: cid, organization_id: organization.id}
+
+    let fact = Object.assign({}, rawFact)
+    _.set(fact, 'simulation.sample.sortedValues', null)
+
+    api(getState()).organizations.addFact(organization, fact, (err, fact) => {
```

Comment:
I don't like how ``fact`` is already defined above.  Can you name that var a bit different?